pkg/kubelet/lifecycle/predicate.go

```diff
@@ -67,6 +67,13 @@ func (w *predicateAdmitHandler) Admit(attrs *PodAdmitAttributes) PodAdmitResult
 	pods := attrs.OtherPods
 	nodeInfo := schedulercache.NewNodeInfo(pods...)
 	nodeInfo.SetNode(node)
+	// Ignore missing extended resources. This is required to support cluster level resources which are not tied to Nodes.
+	// If there happens to be a node level resource that is exposed via device plugin as an extended resource,
+	// it will also be ignored if it is not found in the Node Capacity and Allocatable.
+	// There is an implicit assumption that the lifecycle of device plugins will affect node
+	// health and nodes marked unhealthy should not be accepting new pods and
+	// possibly evicting existing pods too.
+	predicates.RegisterPodFitsResourcesPredicateMetadataProducer(true /* ignoreMissingExtendedResources */)
```

Comment:
Why should it be called from the kubelet side? Another problem is that you are setting it statically anyway. So, this is not better than creating a const in scheduler code that is set to true to ignore missing extended resources. Or maybe I am missing something.

Reply:
The predicate function is shared by scheduler and node, but the `ignoreMissingExtendedResources` flag will be provided from the node only, so we need to have a way to opt-in.